runtime/oti/j9javaaccessflags.h

```diff
@@ -150,13 +150,15 @@
 #define J9_JAVA_CLASS_FINALIZER_CHECK_MASK (J9AccClassFinalizeNeeded | J9AccClassOwnableSynchronizer | J9AccClassContinuation)
 #define J9_JAVA_MODIFIERS_SPECIAL_OBJECT (J9AccClassFinalizeNeeded | J9AccClassReferenceMask)
 
-/* static field helpers*/
+/* static field flags (classAndFlags) */
 #define J9StaticFieldRefBaseType 0x1
 #define J9StaticFieldRefDouble 0x2
 #define J9StaticFieldRefVolatile 0x4
 #define J9StaticFieldRefBoolean 0x8
 #define J9StaticFieldRefPutResolved 0x10
-#define J9StaticFieldRefFlagBits 0x3F
 #define J9StaticFieldRefFinal 0x20
+#define J9StaticFieldIsNullRestricted 0x40
```

Comment:
This flag can be added into the following block:
https://github.com/eclipse-openj9/openj9/blob/40039029f644701a854fff5d76d2ec631413f303/debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/LinearDumper.java#L591

It can be done either in this PR or together with other DDR changes in a separate PR.

Reply:
I'll add this to ddr.

Reply:
I'm not sure what you mean here actually, this section is also using `J9StaticFieldRefFlagBits`.

Reply:
I mean the `J9StaticFieldIsNullRestricted` flag can be turned into string "J9StaticFieldRefBaseType" by `getRegionDetailString()` in DDR when dumping detail information.

Reply:
Typo in the above comment, "J9StaticFieldRefBaseType" should be "J9StaticFieldIsNullRestricted".

Reply:
Okay got it. Thanks